### ISSUE TITLE
Projet M2 EKAP: Natural Language Processing with Disaster Tweets - Mocher Vrignaud

### ISSUE THREAD

**Issue by LucieVrignaud** [JeffAbrahamson/IAE-ann-svm#21]
Membre de l'équipe : Cécile Mocher Lucie Vrignaud

Compétition Kaggle : https://www.kaggle.com/c/nlp-getting-started

Repository : https://github.com/LucieVrignaud/Projet-M2-EKAP-Mocher-Vrignaud.git

**Comment by JeffAbrahamson** [comment]
Pourriez-vous ajouter un README.md ?

**Comment by cecilemch** [comment]
> > Pourriez-vous ajouter un README.md ?
> 
> Bonsoir Monsieur, Le README est bien créé dans notre repository. Je me permets de vous poser une question concernant notre projet et nos données. La variable que nous devons prédire est quantitative continue. Pouvons-nous garder cette variable ainsi et utiliser des techniques de regressions (Regression linaire, Lasso, Ridge et SVR) ou bien transformer cette variable en données binaires, par exemple, afin de faire de la classification ?

@JeffAbrahamson

**Comment by JeffAbrahamson** [comment]
Désolé pour le retard.  (Il faut relancer après deux jours...)

Il faut quelque part utiliser les techniques de SVM et ANN que nous apprenons, mais cela ne vous empêche pas de vous servir d'autres techniques comme la régression également.

Vous pouvez également transformer un problème continu en discret avec un seuil.  Par exemple, vous pourriez modéliser un problème discret avec seuil, puis également modéliser avec régression linéaire où l'extension à un seuil est trivial, ce qui vous permet de comparer la précision des deux.

**Comment by LucieVrignaud** [comment]
@JeffAbrahamson 

Bonjour Monsieur,

Nous avons une erreur que nous n'arrivons pas à résoudre. Lors de la création d'un modèle de réseaux  de neurones nous avons une erreur au moment d'entraîner le modèle.
Après avoir effectuer model.fit nous obtenons une erreur que nous n'arrivons pas à cibler.

Ci-dessous le modèle testé:

model = Sequential()
model.add(Dense(3, input_shape=(2,),activation='relu'))
model.add(Dense(1,activation='sigmoid'))
model.compile(optimizer='adam', loss='binary_crossentropy', metrics = ['accuracy'])

model.fit(X_train,y_train, epochs=20)

**Comment by JeffAbrahamson** [comment]
Je ne sais pas répondre :
* je n'ai pas de contexte, je ne peux pas savoir ce que vous avez compris ou pas, essayé ou pas, quelles conversations vous aviez déjà eu avec les autres étudiants
* vous évoquez une erreur mais vous ne dites pas plus -- je ne suis incapable de la deviner
* je souligne que le premier problème avec ce morceau de code est qu'il n'est qu'un morceau sans contexte -- un lien vers votre code, vers le programme minimal qui reproduit votre erreur pourrait aider.
* pour ce, il conviendrait d'ouvrir un issue sur _votre  repo, la question est lié à votre projet et pas au cour
* d'ailleurs, cet issue est fermé et sur un autre sujet
* rappel (de ce repo) de [bonnes pratiques pour poser des questions](https://github.com/JeffAbrahamson/IAE-ann-svm#questions-hors-s%C3%A9ance)

Je vous invite à mieux formuler votre question et réessayer avec un nouveau issue dans votre repo.

**Comment by LucieVrignaud** [comment]
https://github.com/LucieVrignaud/Projet-M2-EKAP-Mocher-Vrignaud.git

Nous avons terminé notre rapport et notre code que vous trouverez dans notre repo ci-dessus.
